Lib/tempfile.py

```diff
@@ -743,18 +755,21 @@ def tell(self):
 
     def truncate(self, size=None):
         if size is None:
-            self._file.truncate()
+            return self._file.truncate()
         else:
```

Comment:
this else is not needed, PEP warning, might as well fix

Reply:
It's not necessary to remove the else.  It does no harm.

Reply:
I never said it was necessary, just bad style as it unnecessarily increased the code depth, take my comment as an FYI

Reply:
Just wanted to confirm @merwok - nothing needs to be changed here for approval?

Reply:
Right, I advise to not change the `else` line.